### PR TITLE
scale deeptools with n samples

### PR DESCRIPTION
**Pull request by siebrenf**
**What problem is the PR solving**
with a large number of samples, this plot is very artistic, yet slightly less useful...
image

**What did change**
the size of the plot now scales with the number of bams. The new plot may not be any more readable in the MultiQC itself, but if the pixel density remains the same, you can now open it in a separate window and zoom in (similar to the DESeq2 heatmaps).

**Checklist**
- [x] I made a PR to develop (not master)
- [ ] If applicable: I updated the docs
- [ ] I updated the CHANGELOG
- [ ] These changes are covered by the tests

### CODE REVIEW

**Comment by siebrenf** [comment]
The new figures can get too small to read in the MultiQC, but you can open them separately. 
P.S. the whitespace and DPI cannot be fixed.

0 30 GRCz11-deepTools_spearman_correlation_clustering_mqc

File: seq2science/rules/qc.smk [hunk 1]
```diff
@@ -494,12 +494,25 @@ rule multiBamSummary:
...
+
+
 rule plotCorrelation:
     """
     Calculate the correlation between bams with deepTools.
     """
     input:
-        rules.multiBamSummary.output,
+        unpack(fingerprint_multiBamSummary_input),
+        cordata=rules.multiBamSummary.output,
```
Comment (Maarten-vd-Sande) [inline seq2science/rules/qc.smk:515]: Is this necessary?

**Review by Maarten-vd-Sande — APPROVED** [review]
Awesome!

File: seq2science/rules/qc.smk (hunk 1 above)
Comment (siebrenf) [reply seq2science/rules/qc.smk:515]: `fingerprint_multiBamSummary_input` contains a list of bam files, which is used in `get_plotCor_opts()` to determine the number of samples. 

This seemed like the easiest source to me. If you have a better idea I can change it!

Comment (Maarten-vd-Sande) [reply seq2science/rules/qc.smk:515]: ah like that. Sure makes sense! :+1:

**Pull request merged**